tensorflow_model_optimization/g3doc/guide/clustering/clustering_example.ipynb

```diff
@@ -0,0 +1,515 @@
+{
```

Comment:
<p>The conclusion can be stronger by referencing the numbers again.</p><p> </p><p>"We showed you" -&gt; "You created clustered models" according to you/we in style guide. <a href="https://www.google.com/url?sa=D&amp;q=https%3A%2F%2Fdevelopers.google.com%2Fstyle%2Fperson" target="_blank">https://developers.google.com/style/person</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/model-optimization/pull/330/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>